applications/asset_tracker_v2/prj.conf

```diff
@@ -99,15 +92,32 @@ CONFIG_MODEM_INFO=y
 # nRF Cloud
 CONFIG_NRF_CLOUD=y
 CONFIG_NRF_CLOUD_MQTT=y
-CONFIG_AGPS=y
-CONFIG_AGPS_SRC_NRF_CLOUD=y
-CONFIG_NRF_CLOUD_AGPS=y
-## Because PGPS consumes a lot of data its disabled by default.
-CONFIG_NRF_CLOUD_PGPS=n
 CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD=y
 CONFIG_NRF_CLOUD_SEND_TIMEOUT=y
+
+# GPS
 CONFIG_GPS_MODULE_NMEA=y
 
+## A-GPS and P-GPS
```

Comment:
I moved around a bit on these configs and grouped GPS related ones together. Clean-up of commented-out configs and moving them to separte config files to be done in separate PR.